cmd/agola/cmd/completionbash.go

```diff
@@ -0,0 +1,47 @@
+// Copyright 2019 Sorint.lab
+//
+// Licensed under the Apache License, Version 2.0 (the "License");
+// you may not use this file except in compliance with the License.
+// You may obtain a copy of the License at
+//
+//     http://www.apache.org/licenses/LICENSE-2.0
+//
+// Unless required by applicable law or agreed to in writing, software
+// distributed under the License is distributed on an "AS IS" BASIS,
+// WITHOUT WARRANTIES OR CONDITIONS OF ANY KIND, either express or implied
+// See the License for the specific language governing permissions and
+// limitations under the License.
+
+package cmd
+
+import (
+	"github.com/spf13/cobra"
+)
+
+var cmdCompletionBash = &cobra.Command{
+	Use: "bash",
+	Run: func(cmd *cobra.Command, args []string) {
+		if err := completionShell(cmd, args, "bash"); err != nil {
+			log.Fatalf("err: %v", err)
+		}
+	},
+	Short: "generates bash completion scripts",
+	Long: `generates bash completion scripts
+	
+To load bash completion in the current session run:
+
+  source <(agola completion bash)
+
+To configure your bash shell to load completions for each session add the scripts to your ~/.bashrc file:
```

Comment:
We should specify the the agola command must be in the user command search paths (directories specified in the PATH env var) or change it to the absolute path of the agola binary.

Reply:
Added a note at the end of the message:
```
  NOTE: the agola command must be in the user command search paths (PATH environment variable) or it is necessary to specify the absolute path of the agola binary (e.g. /your/path/agola).
```